airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -410,11 +410,15 @@ def __init__(
         *,
         cluster_identifier: str,
         aws_conn_id: str = "aws_default",
+        deferrable: bool = False,
+        poll_interval: int = 10,
```

Comment:
Can you include these in the docstring please?

Reply:
fixed